custom_components/alexa_media/sensor.py

```diff
@@ -226,6 +261,80 @@ def unique_id(self):
         return self.alexa_entity_id + "_temperature"
 
 
+class AirQualitySensor(SensorEntity, CoordinatorEntity):
+    """A air quality sensor reported by an Amazon indoor air quality monitor."""
+
+    def __init__(self, coordinator, entity_id, name, media_player_device_id, sensor_name, instance, unit):
+        super().__init__(coordinator)
+        self.alexa_entity_id = entity_id
+        self._sensor_name = sensor_name
+        # tidy up name
+        self._sensor_name = self._sensor_name.replace("Alexa.AirQuality.",'')
+        self._sensor_name = ''.join(' ' + char if char.isupper() else char.strip() for char in self._sensor_name).strip()
+        self._name = name + " " + self._sensor_name
+        self._media_player_device_id = media_player_device_id
+        
+        self._instance = instance
+        
+        match unit:
```

Comment:
I normally like using dicts instead of match statements so the code is data driven by updating some values in const.py. What's your thoughts here?

Reply:
I hadnt considered using dicts for this. It would certainly make the code tidier.

Reply:
Do you mind making that change?

Reply:
Nevermind, I handled it. Also I realized that match isn't available in python 3.9 which we still support.